README.md

```diff
@@ -54,275 +55,250 @@ limitations under the License.
+- This library is not a modular toolbox of building blocks for neural nets. The code in the model files is not refactored with additional abstractions on purpose, so that researchers can quickly iterate on each of the models without diving into additional abstractions/files.
+- The training API is optimized to work with PyTorch models provided by Transformers. For generic machine learning loops, you should use another library like [Accelerate](https://huggingface.co/docs/accelerate).
+- The [example scripts]((https://github.com/huggingface/transformers/tree/main/examples)) are only *examples*. They may not necessarily work out-of-the-box on your specific use case and you'll need to adapt the code for it to work.
 
-When one of those backends has been installed, 🤗 Transformers can be installed using pip as follows:
+## 100 projects using Transformers
 
-```
-pip install transformers
-```
+Transformers is more than a toolkit to use pretrained models, it's a community of projects built around it and the
+Hugging Face Hub. We want Transformers to enable developers, researchers, students, professors, engineers, and anyone
+else to build their dream projects.
 
-If you'd like to play with the examples or need the bleeding edge of the code and can't wait for a new release, you must [install the library from source](https://huggingface.co/docs/transformers/installation#installing-from-source).
+In order to celebrate Transformers 100,000 stars, we wanted to put the spotlight on the
+community with the [awesome-transformers](./awesome-transformers.md) page which lists 100
+incredible projects built with Transformers.
 
-```
-git clone https://github.com/huggingface/transformers.git
-cd transformers
-pip install .
-```
+If you own or use a project that you believe should be part of the list, please open a PR to add it!
 
-### With conda
+## Example models
 
-🤗 Transformers can be installed using conda as follows:
+You can test most of our models directly on their [Hub model pages](https://huggingface.co/models).
 
-```shell script
-conda install conda-forge::transformers
-```
+Expand each modality below to see a few example models for various use cases.
 
-> **_NOTE:_** Installing `transformers` from the `huggingface` channel is deprecated.
+<details>
+<summary>Audio</summary>
 
-Follow the installation pages of Flax, PyTorch or TensorFlow to see how to install them with conda.
+- Audio classification with [Whisper](https://huggingface.co/openai/whisper-large-v3-turbo)
+- Automatic speech recognition with [Moonshine](https://huggingface.co/UsefulSensors/moonshine)
+- Keyword spotting with [Wav2Vec2](https://huggingface.co/superb/wav2vec2-base-superb-ks)
+- Speech to speech generation with [Moshi](https://huggingface.co/kyutai/moshiko-pytorch-bf16)
+- Text to audio with [MusicGen](https://huggingface.co/facebook/musicgen-large)
+- Text to speech with [Bark](https://huggingface.co/suno/bark)
```

Comment:
cc @eustlb @Vaibhavs10 in case you see better/others that would fit here

Reply:
gentle ping @eustlb @Vaibhavs10 if there are better audio models to add here :)